vsphere/datadog_checks/vsphere/vsphere.py

```diff
@@ -957,10 +718,14 @@ def check(self, instance):
                 thread_crashed = True
         except Empty:
             pass
+
         if thread_crashed:
             self.stop_pool()
             raise Exception("One thread in the pool crashed, check the logs")
 
-        ### <TEST-INSTRUMENTATION>
+        if set_external_tags is not None:
+            set_external_tags(self.get_external_host_tags())
```

Comment:
As discussed this may be a problem due to the granularity with which we submit the host metadata payload from agent6. More specifically, the host tags are susceptible to change within the submission window - I believe the current behavior would be very open to inaccurate reporting of tags associated to the VMs.  

This will be fine on agent5.

Reply:
Working on a change to send out the tags every 10 minutes, will be in 6.0.0-rc2